terraform/transform_transitive_reduction.go

```diff
@@ -0,0 +1,20 @@
+package terraform
+
+// TransitiveReductionTransformer is a GraphTransformer that performs
+// finds the transitive reduction of the graph. For a definition of
+// transitive reduction, see Wikipedia.
```

Comment:
Hahahah - did you just http://www.letmewikipediathatforyou.com/?q=Transitive%20Reduction in a code comment?